packages/vite/src/node/server/index.ts

```diff
@@ -313,6 +313,8 @@ export interface ResolvedServerUrls {
   network: string[]
 }
 
+const defaultPort = 5173
```

Comment:
I think we should do for both ports
As far as I know we have the DEV_PORT = 5173 and the PREVIEW_PORT(/SERVE_PORT?) = 4173
Please check this before

IMO we could then export these constants so external could even use it
Other npm packages e.g. export a VERSION. Here is just one example out of my mind: https://github.com/vladmandic/face-api/blob/d36ed6d266140c843d51b4ecada383f02542e550/src/index.ts#L25

Reply:
At the end I mean to have something like:
```ts
import { DEFAULT_DEV_PORT, DEFAULT_PREVIEW_PORT } from 'vite';
```